### ISSUE TITLE
xWebDeploy SourcePath throw exception with spaces

### ISSUE THREAD

**Issue by MKletz** [PowerShell/xWebDeploy#28]
Not sure if this is my fault or a bug but when providing a path to a zip that contains spaces an exception is thrown.

Example "\\server\share\website deployment\website.zip"

The stack overflow thread below implies it's a syntax issue for how msdeploy is being passed arguments.

https://stackoverflow.com/questions/3499699/how-do-you-call-msdeploy-from-powershell-when-the-parameters-have-spaces

**Comment by johlju** [comment]
Yes, probably a case where the path is not enclosed in quotes. Sounds like a bug.

https://github.com/PowerShell/xWebDeploy/blob/1e862c668b943923b560b047ecd53c384182a9ae/DSCResources/xWebPackageDeploy/xWebPackageDeploy.psm1#L94-L105
